src/main/java/com/uid2/admin/vertx/service/SaltService.java

```diff
@@ -61,6 +61,7 @@ private void handleSaltSnapshots(RoutingContext rc) {
                     .putHeader(HttpHeaders.CONTENT_TYPE, "application/json")
                     .end(ja.encode());
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
nitpick - do you still want these printStackTrace() above/below, or do `LOGGER.error(e.getMessage(), e); `instead?